plugins/modules/apk.py

```diff
@@ -35,7 +35,7 @@
     default: false
   name:
     description:
-      - A package name, like V(foo), or multiple packages, like V(foo, bar).
+      - A package name, like V(foo), or multiple packages, like V(foo,bar). Do not include additional whitespace when specifying multiple packages.
```

Comment:
How about making this a new paragraph, and hinting at YAML lists
```suggestion
      - A package name, like V(foo), or multiple packages, like V(foo,bar).
      - Do not include additional whitespace when specifying multiple packages as a string.
        Prefer YAML lists over comma-separating multiple package names.
```